### PR TITLE
Feature/sumin#index#001

### PR DESCRIPTION
**Pull request by blueblue3824**
주석추가, index.jsp 코드정리

### CODE REVIEW

**Review by dpoowa — COMMENTED** [review]
commit OK

**Review by dpoowa — COMMENTED** [review]
okok

**Review by yooninkwon — COMMENTED** [review]
ok

File: src/main/java/com/tech/petfriends/index/controller/MainController.java
Comment (dpoowa) [inline src/main/java/com/tech/petfriends/index/controller/MainController.java:1]: ok

File: src/main/resources/application.properties
Comment (dpoowa) [inline src/main/resources/application.properties:1]: ok

**Review by dpoowa — APPROVED** [review]
OK

**Pull request merged**